### DISCUSSION TITLE
Julia support

### DISCUSSION THREAD

**Discussion by MarkusZechner** [lckr/jupyterlab-variableInspector#199] [Ideas]
Hey!

Thanks a lot for this variable inspector - very helpful!!

Are there any plans to also support Julia?

Thanks a lot!

Markus

**Comment by lckr** [comment]
Hey Markus,
Someone already implemented parts of the necessary functionality in #133. There are still some things missing, but if you can live without this features it should be working as far as I know.
If you're more familiar with Julia you also might look at adding these features ;-)
